src/app/containers/StoryPromo/index.jsx

```diff
@@ -87,6 +93,7 @@ const StoryPromoContainer = ({
   displayImage,
   displaySummary,
   isRecommendation,
+  isFullColumn,
```

Comment:
Maybe a better name would be something along the lines of `isSingleColumnLayout`?

Reply:
Done